docs/docs/migration/migration-from-v6-to-v7.md

```diff
@@ -8,8 +8,86 @@ sidebar_position: 1
 First, execute `npx @comet/upgrade@latest v7` in the root of your project.
 It automatically installs the new versions of all `@comet` libraries, runs an ESLint autofix and handles some of the necessary renames.
 
+<details>
+
+<summary>Changes handled by @comet/upgrade</summary>
+
+-   Disabling GraphQL field suggestions
+-   Importing the types of `@comet/admin-theme` in `vendors.d.ts`
+-   Replacing the Roboto font with Roboto Flex
+
+</details>
+
 ## API
 
+### Remove unnecessary dependencies
+
+Remove following dependencies **if you don't use them in your project**:
```

Comment:
Maybe add a note here that those dependencies were previously peer dependencies?

Reply:
bc776049c9eda8a52af3196e038e67ad95ec6f23